package.json

```diff
@@ -303,6 +303,11 @@
           "default": 2,
           "markdownDescription": "How many levels to indent the attributes by when `#xml.format.splitAttributes#` is `true`. Default value is `2`. See [here](command:xml.open.docs?%5B%7B%22page%22%3A%22Formatting%22%2C%22section%22%3A%22xmlformatsplitattributesindentsize%22%7D%5D) for more information"
         },
+        "xml.format.closingBracketNewLine": {
+          "type": "boolean",
+          "default": false,
+          "markdownDescription": "The option to put a closing tag on a newline when `#xml.format.splitAttributes#` is `true`. Default value is `true` See [here]() for more information."
```

Comment:
Add an entry to `docs/Formatting.md` that explains what this setting does, with a short example/demo (it can just be a plain text example like the other settings). Then link to the documentaion in the "markdownDescription"